app/components/thumbnail.jsx

```diff
@@ -47,9 +47,10 @@ export default class Thumbnail extends React.Component {
     };
 
     if (this.props.format === 'mp4') {
+      const width = this.props.width < 999 ? this.props.width : '300';
       return (
```

Comment:
I don't think you should be overriding the default width here. If the default value isn't sensible, then change that instead.